--- OMERO.py-5.4.10-ice36-b105.orig/lib/python/omeroweb/webgateway/marshal.py	2019-01-24 12:56:12.000000000 +0000
+++ OMERO.py-5.4.10-ice36-b105/lib/python/omeroweb/webgateway/marshal.py	2019-02-14 18:10:55.000000000 +0000
@@ -53,6 +53,10 @@
             if (hasattr(event_context, a)):
                 ctx[a] = getattr(event_context, a)
 
+    for a in ['sudoerId', 'sudoerName']:
+            if (hasattr(event_context, a)):
+                ctx[a] = unwrap(getattr(event_context, a))
+
     perms = event_context.groupPermissions
     encoder = get_encoder(perms.__class__)
     ctx['groupPermissions'] = encoder.encode(perms)
